test.sh

```diff
@@ -61,32 +62,32 @@ then
 	fi
 
 	echo "> Creating images"
-	dd if=/dev/zero of=block count=10 bs=1M 2>&1 | sed 's/^/\t/g'
+	dd if=/dev/zero of=block count=10 bs=1M 2>&1 | sed 's/^/\t/'
 
 	echo "> Setting up the loop devices"
 	block="$(sudo losetup -f)"
 	echo "> Setting up block as $block"
-	sudo losetup $block block 2>&1 | sed 's/^/\t/g'
+	sudo losetup $block block 2>&1 | sed 's/^/\t/'
 
 	echo "> Formatting '$block' with passphrase '$passphrase'."
-	printf "%s" "$passphrase" | sudo cryptsetup luksFormat -q $block - 2>&1 | sed 's/^/\t/g'
+	printf "%s" "$passphrase" | sudo cryptsetup luksFormat -q $block - 2>&1 | sed 's/^/\t/'
 	echo "> Opening the device '$block' as /dev/mapper/temp-test"
-	printf "%s" "$passphrase" | sudo cryptsetup luksOpen -q $block temp-test - 2>&1 | sed 's/^/\t/g'
+	printf "%s" "$passphrase" | sudo cryptsetup open -q $block temp-test - 2>&1 | sed 's/^/\t/'
 	echo "> Creating a filesystem on '/dev/mapper/temp-test'"
 	sudo mkfs.ext2 /dev/mapper/temp-test
 	echo "> Mounting the fs"
 	sudo mount -t ext2 /dev/mapper/temp-test local-mount
 	echo "> Creating proof in the mounted fs"
-	sudo sh -c 'date "+proof:%s" > local-mount/proof'
-	proof=$(cat local-mount/proof)
+	proof=$(date "+proof:%s" | sudo tee local-mount/proof)
 	echo "> Proof is: '$proof'"
 	echo "> Unmounting the fs"
 	sudo umount local-mount
 	echo "> Closing the device '/dev/mapper/temp-test'"
-	sudo cryptsetup luksClose temp-test
+	sudo cryptsetup close temp-test
```

Comment:
Unlike the `open --type luks` syntax, this is the correct one (the man documents it as the command `luksClose` is aliased to). :+1: